events/event_queue.go

```diff
@@ -114,7 +119,8 @@ func StartConsumers(gormDB *gorm.DB, pgpool *pgxpool.Pool, config Config) {
 			uniqEvents[event] = struct{}{}
 		}
 
-		go allSyncHandlers[i].EventConsumer(gormDB, pgpool).Listen()
+		pgNotifyChannel := notifyRouter.RegisterRoutes(allSyncHandlers[i].watchEvents)
```

Comment:
Is this the pg notify channel or a notification channel

Reply:
pg notify channel but distributed by the NotifyRouter for the given watch events.